### ISSUE TITLE
Show nn per page change and Apply to ALL tables doesn't work in Global Search

### ISSUE THREAD

**Issue by rwill1** [PATRIC3/patric3_website#156]
In Global Search, search for TB-ARC Mali, Click on Genomes (99), scroll to the bottom and change Show 20 to Show 99 click Apply to ALL tables (and 99 search results show up).
Nexst Global Search for TB-ARC KRITH, Click on Genomes (399) and only 20 genomes are displayed

**Comment by mshukla1** [comment]
Global Search results are NOT A TABLE...you didn't get the memo?!! ;)

**Comment by mshukla1** [comment]
It seems to be working for me, if I use Apply All Tables on mac/chrome. Let me know if you still have the problem.

**Comment by rwill1** [comment]
still a problem for me.   I just tried it as described above, using Mac / Safari, cache cleared.   It works the same for me on Production and Beta.   I suspect that this happened when we switched from old workspace to new workspace.   I also suspect that it's working for Maulik because he's logged in.... just a guess.   Since the behavior is the same on beta and production, I don't think this is a required for release.
